src/index.html

```diff
@@ -1,32 +1,132 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Recipe finder</title>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <title>Recipe finder</title>
 </head>
+
 <body>
-    <header> <!--START OF NAV BAR-->
-        <img src="./assets/images/logoRecipeEasy.svg" alt="Logo">
-        <h1>Recipe Easy</h1>
-        <input type="text" name="search-bar" value="I want to eat..">
-        <button>Search</button>
-        <button>Can´t decide?</button>
-     </header> <!--END OF NAV BAR-->
+   <header>
+      <!--START OF NAV BAR-->
+      <img src="./assets/images/logoRecipeEasy.svg" alt="Logo">
+      <h1>Recipe Easy</h1>
+      <input type="text" name="search-bar" value="I want to eat..">
+      <button>Search</button>
+      <button>Can´t decide?</button>
+   </header>
+   <!--END OF NAV BAR-->
+
+   <!--START OF MAIN SECTION-->
+   <section class="meals">
+      <h1 class="title">MEALS</h1>
```

Comment:
We must put only one h1 tag for each page, so I think this text(title) could be an h2 tag. Maybe we can use the h1 tag for another more important text on the page like the slogan.

Reply:
Ohh ok. Thanks!! Changes have been made.